.github/dependabot.yml

```diff
@@ -0,0 +1,15 @@
+# To get started with Dependabot version updates, you'll need to specify which
+# package ecosystems to update and where the package manifests are located.
+# Please see the documentation for all configuration options:
+# https://help.github.com/github/administering-a-repository/configuration-options-for-dependency-updates
+
+version: 2
+updates:
+  - package-ecosystem: "npm" # See documentation for possible values
+    directory: "/" # Location of package manifests
+    schedule:
+      interval: "daily"
+    versioning-strategy: lockfile-only
+    commit-message:
+      prefix: "npm"
+      include: "scope"
```

Comment:
```suggestion
      include: "scope"
    allow:  # Only consider run time dependencies:
      - dependency-type: "production"
```

Reply:
Thanks for this! I read the docs listed, and got the impression that this change would be needed?

Reply:
We could indeed reduce the number of notifications to target only the production (aka those that matters).

Side note: I turn it on on some [other projects](https://github.com/jupyterlab-contrib/jupyterlab_toastify/pulls?q=is%3Apr+is%3Aclosed+bump). Unfortunately it seems to trigger far more bump than the default set up for security analysis. Source I found out:

- https://github.community/t/how-to-get-dependabot-to-trigger-for-security-updates-only/117257
- https://stackoverflow.com/questions/64047526/how-to-get-dependabot-to-trigger-for-security-updates-only

So this sounds like half satisfactory. What do you think?